applications/FluidDynamicsApplication/python_scripts/navier_stokes_ale_fluid_solver.py

```diff
@@ -4,9 +4,13 @@
 import KratosMultiphysics
 
 # other imports
-from KratosMultiphysics.MeshMovingApplication.ale_fluid_solver import AleFluidSolver
 import KratosMultiphysics.FluidDynamicsApplication.python_solvers_wrapper_fluid as fluid_solvers_wrapper
 
+try:
+    from KratosMultiphysics.MeshMovingApplication.ale_fluid_solver import AleFluidSolver
+except Exception:
+    raise("Unable to load MeshMovingApplication")
+
```

Comment:
I added a ward here so it cannot work if MeshMoving is not compiled

Reply:
I would go with kratos_utilities/is_app_available (don't remember the correct name) to avoid the attempted import